app/src/main/java/org/eyeseetea/malariacare/domain/entity/Country.java

```diff
@@ -1,24 +1,17 @@
 package org.eyeseetea.malariacare.domain.entity;
 
 
+import static org.eyeseetea.malariacare.domain.utils.RequiredChecker.required;
+
 public class Country {
     private long id;
     private String uid;
     private String name;
 
     public Country(long id, String uid, String name) {
-        this.id = id;
-        this.uid = uid;
-        this.name = name;
-    }
-
-    public Country() {
-    }
-
-    private Country(Builder builder) {
```

Comment:
Can you remove private setters? It's not necessary we have private setters, so we reduce the size of the class

Reply:
Done!